components/collector/__init__.py

```diff
@@ -67,8 +66,9 @@ def __post_init__(self):
         # We want these as Python lists because the entries will have different lengths.
         self.utc_times = [t.to_datetime(pytz.UTC) for t in self.times]
         self.local_times = [t.to_datetime(self.site.value.timezone) for t in self.times]
-        self.local_sidereal_times = [vskyutil.lpsidereal(t, self.site.value.location) for t in self.times]
-
+        # self.local_sidereal_times = [vskyutil.lpsidereal(t, self.site.value.location) for t in self.times]
```

Comment:
Thanks for leaving these in so I can check. I'll take them out once I convince myself everything looks right but so far, browsing, things look fine.